auditors/all/all.go

```diff
@@ -57,6 +52,17 @@ func Auditors(conf config.KubeauditConfig) ([]kubeaudit.Auditable, error) {
 	return auditors, nil
 }
 
+// getEnabledAuditors returns a list of all auditors excluding any explicitly disabled in the config
+func getEnabledAuditors(conf config.KubeauditConfig) []string {
+	auditors := []string{}
+	for _, auditorName := range AuditorNames {
+		if enabled, ok := conf.GetEnabledAuditors()[auditorName]; !ok || enabled {
```

Comment:
maybe we can add a comment here to explain this line(_enabled, ok := conf.GetEnabledAuditors()[auditorName]; !ok || enabled_)?
```
// if value is not found in the `conf.GetEnabledAuditors()` map, this means
// it wasn't added to the config file, so it should be enabled by default
```

another option is to rename the var and instead of calling it enabled, call it `explicitlyEnabledInConfig`?